pkgs/unified_analytics/test/log_handler_test.dart

```diff
@@ -290,3 +334,51 @@ void main() {
     expect(newString, testString);
   });
 }
+
+class _FakeFileStat extends Fake implements FileStat {
+  _FakeFileStat(this.size);
+
+  @override
+  final int size;
+}
+
+class _FakeFile extends Fake implements File {
+  _FakeFile(this.path);
```

Comment:
Out of curiosity, why use mutable fields for method implementations over optional constructor parameters? Do you find this more readable?